consumer/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maven:3.8.4-openjdk-17-slim AS build
+COPY src /home/app/src
+COPY pom.xml /home/app
+RUN mvn -f /home/app/pom.xml clean package
+
+ENV PULSAR_BROKER_URL='broker-1:6650,broker-2:6651'
+ENV TOPIC_NAME = 'Testtopic'
+ENV SUBSCRIPTION_NAME = 'subscription'
```

Comment:
Should we allow the user to configure more of the consumer properties through environments?